src/include/time_inline.h

```diff
@@ -48,20 +48,20 @@ __wt_rdtsc(void)
  *     highest value seen so far.
  */
 static inline void
-__time_check_monotonic(WT_SESSION_IMPL *session, struct timespec *tsp)
+__time_check_monotonic(WT_SESSION_METADATA *meta, struct timespec *tsp)
```

Comment:
This was where it all started - I picked this function and started thinking about "what would it take to remove the session passed in here?"